### PR TITLE
Assorted webpack related updates

### PR DESCRIPTION
**Pull request by cquiroz**
This PR updates a few npm libs and improves the sbt build. I'm opening the PR to test also deploying from heroku

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -194,28 +194,28 @@ lazy val commonJsLibSettings = gspScalaJsSettings ++ commonLibSettings ++ Seq(
 )
 
 lazy val commonWDS = Seq(
-  version in webpack := "4.43.0",
-  version in startWebpackDevServer := "3.11.0",
+  webpack / version := "4.43.0",
+  startWebpackDevServer / version := "3.11.0",
   webpackConfigFile in fastOptJS := Some(
```
Comment (rpiaggio) [inline build.sbt:199]: Should we change this to fastOptJS / webpackConfigFile too? (And the same for fullOptJS?)

File: build.sbt [hunk 2]
```diff
@@ -194,28 +194,28 @@ lazy val commonJsLibSettings = gspScalaJsSettings ++ commonLibSettings ++ Seq(
...
   webpackMonitoredFiles / includeFilter := "*",
   useYarn := true,
-  webpackBundlingMode in fastOptJS := BundlingMode.LibraryOnly(),
-  webpackBundlingMode in fullOptJS := BundlingMode.Application,
+  fastOptJS / webpackBundlingMode := BundlingMode.LibraryOnly(),
+  fullOptJS / webpackBundlingMode := BundlingMode.Application,
   test := {},
-  scalaJSLinkerConfig in (Compile, fastOptJS) ~= { _.withSourceMap(false) },
-  scalaJSLinkerConfig in (Compile, fullOptJS) ~= { _.withSourceMap(false) },
+  Compile / fastOptJS / scalaJSLinkerConfig ~= { _.withSourceMap(false) },
```
Comment (rpiaggio) [inline build.sbt:213]: Does fastOptJS / Compile / scalaJSLinkerConfig work too? If it does, I find it easier to process when glancing over the file quickly.

File: build.sbt (hunk 1 above)
Comment (cquiroz) [reply build.sbt:199]: Great catch

File: build.sbt (hunk 2 above)
Comment (cquiroz) [reply build.sbt:213]: The order is quite specific

**Pull request merged**